src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/UserTests.kt

```diff
@@ -23,6 +23,48 @@ class UserTests : DatabaseTest()
             it.addUserWithRoles("testuser",
                     ReifiedRole("member", Scope.Specific("modelling-group", "group")),
                     ReifiedRole("user", Scope.Global()))
+        } requiringPermissions {
+            PermissionSet("*/users.read")
+        } andCheck {
+            Assertions.assertThat(it).isEqualTo(json {
+                obj(
+                        "username" to "testuser",
+                        "name" to "Test User",
+                        "email" to "testuser@example.com",
+                        "last_logged_in" to null
+                )
+            })
+        }
+    }
+
+    @Test
+    fun `cannot get user without user reading permission`()
```

Comment:
We don't need this test, as the fluent validation helper in the test above will use the `requiringPermissions` stanza and will automatically check that all those permissions are required. To prove this to yourself, remove the required permission from the controller and try running the test above.